os/drivers/compression/compress.c

```diff
@@ -120,12 +120,12 @@ static int comp_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 			return -EINVAL;
 		}
 		switch (CONFIG_COMPRESSION_TYPE) {
-			case LZMA_TYPE:
```

Comment:
Please add folder path in commit title. 
Change the commit title to : Fix indentation issues